pytorch_lightning/accelerators/accelerator.py

```diff
@@ -409,7 +415,7 @@ def process_dataloader(self, dataloader: Union[Iterable, DataLoader]) -> Union[I
     @property
     def results(self) -> Any:
         """
-        The results of the last training/testing run will be cached here.
+        The results of the last training/testing run will be cached within the training type plugin.
         In distributed training, we make sure to transfer the results to the appropriate master process.
         """
         # TODO: improve these docs
```

Comment:
Improved enough already to remove this?

Reply:
I don't like rando todos, so let me remove it :)

Reply:
my comment, but I don't remember what I wanted to improve :rofl: